app/assets/javascripts/discourse/app/components/composer-title.js

```diff
@@ -153,11 +153,17 @@ export default Component.extend({
       this.set("composer.featuredLink", this.get("composer.title"));
 
       const $h = $(html),
-        heading = $h.find("h3").length > 0 ? $h.find("h3") : $h.find("h4"),
         composer = this.composer;
 
       composer.appendText(this.get("composer.title"), null, { block: true });
 
+      if ($h.hasClass("twitterstatus")) {
```

Comment:
This might be a good opportunity to replace this with regular JS, since the jQuery here is doing nothing special.